decidim-debates/app/presenters/decidim/debates/debate_presenter.rb

```diff
@@ -33,6 +33,39 @@ def description(strip_tags: false)
         content = strip_tags(content) if strip_tags
         content
       end
+
+      def last_comment_by
+        return unless comments_authors.any?
+
+        comment_author = comments.order("created_at DESC").first.normalized_author
+        return unless comment_author
+
+        if comment_author.is_a?(Decidim::User)
+          Decidim::UserPresenter.new(comment_author)
+        elsif comment_author.is_a?(Decidim::UserGroup)
+          Decidim::UserGroupPresenter.new(comment_author)
+        elsif comment_author.is_a?(Decidim::Organization)
+          Decidim::Debates::OfficialAuthorPresenter.new
+        end
```

Comment:
maybe we can simplify this lines to:
```suggestion
        comment_author.presenter
```

if `Decidim::Organization` includes `Decidim::ActsAsActor` and implements the `presenter` method.
Can you change it please?